.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
```

Comment:
**[Optional]**

- [ ] I am not able to logout from your application 😄 Kindly make a button there that would log the user out 👍 and also make your application professional 😊